### DISCUSSION TITLE
Should `widget.gtk.non-native-titlebar-buttons.enabled` be set to false by default?

### DISCUSSION THREAD

**Discussion by uncompute** [zen-browser/desktop#2521] [Ideas]
On GNOME, if you set `widget.gtk.non-native-titlebar-buttons.enabled` to false the close button will be in style with the rest of the system

When `true` is set
![true](https://github.com/user-attachments/assets/bc57b638-618c-43fa-8b6e-d88faff46f62)
When `false` is set
![false](https://github.com/user-attachments/assets/11e7dc60-4b10-42b8-8e18-cc5d5e4800de)

**Comment by BenjaminPalko** [comment]
Thank you for asking this, I couldn't figure out why my titlebar buttons were so f'd up and this solved it!
